client/src/components/Navbar.jsx

```diff
@@ -472,7 +472,7 @@ const MobileNavItem = ({ label, children, href }) => {
                               modalComponentStore.closeModal();
                               redirectToSessionPage(data);
                               toast({
-                                title: `Successfully matched with User #${data.uid} on ${data.complexity} question - ${data.title}`,
+                                title: `Successfully matched with User #${data.firstUserId} on ${data.complexity} question - ${data.title}`,
```

Comment:
@changjy44 I think the socket messages to the 2 clients will still have to be different, since the clients don't know who's first and second

Reply:
I disagree. I think we should just define user1 as the first user, and the user2 as the second user. Passing the entire session object for both parties is cleaner in case we want to extend our functionality

Reply:
Okay, then could you fix the bug on the FE? Coz both users see the same notification message rn